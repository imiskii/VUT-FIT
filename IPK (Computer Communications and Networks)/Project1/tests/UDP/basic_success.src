(+ 5 7)
(- 2 10)
(* 8 3)
(* 0 2)
(/ 64 8)