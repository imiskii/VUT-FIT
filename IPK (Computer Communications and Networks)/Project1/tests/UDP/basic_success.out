OK:12
OK:-8
OK:24
OK:0
OK:8

